src/transpilers/swc.ts

```diff
@@ -225,14 +225,15 @@ export function createSwcOptions(
         parser: {
           syntax: 'typescript',
           tsx: isTsx,
-          decorators: experimentalDecorators,
+          decorators: true,
           dynamicImport: true,
           importAssertions: true,
         } as swcWasm.TsParserConfig,
         target: swcTarget as swcWasm.JscTarget,
         transform: {
           decoratorMetadata: emitDecoratorMetadata,
-          legacyDecorator: true,
+          legacyDecorator: experimentalDecorators,
+          decoratorVersion: experimentalDecorators ? '2021-12' : '2022-03',
```

Comment:
```suggestion
          decoratorVersion: experimentalDecorators ? '2021-12' : '2023-11',
```

Reply:
> Where did you get that 2023-11 from? SWC doesn't have that option: https://swc.rs/docs/configuration/compilation#jsctransformdecoratorversion

I saw this value in SWC error message & Babel document, maybe it hasn't been implemented.

Reply:
no it hasn't